lib/puppet/provider/gcompute_url_map/google.rb

```diff
@@ -134,8 +134,7 @@ def dirty(field, from, to)
 
   def exports
     {
-      self_link: @fetched['selfLink'],
-      project: resource[:project]
```

Comment:
Why are all these projects suddenly safe to remove?  They were only used for self link construction?

Reply:
Yep!

Quick check for my sanity: `git blame` says the line creating those exports points to commit [51cb9762184f6f525c67d99e374f10bfd3e510b3](https://github.com/GoogleCloudPlatform/magic-modules/commit/51cb9762184f6f525c67d99e374f10bfd3e510b3)

Reply:
Great!  Then all good, go ahead and merge all these child PRs and the main one.